Exercise_B/index.css

```diff
@@ -1,4 +1,20 @@
-/* 
-This css sheet is linked to index.html
-You can style tags, classes (.) or ids (#) from this file.
-*/
+.red{
+    background-color: red;
+    margin: 0 auto;
+    clear: both;
```

Comment:
Fantastic use of float and clear!
Nice use of margin to get the red div into the center
As red, blue and green are technically unique as no where else uses the classes consider using id